numpy/_core/src/multiarray/stringdtype/utf8_utils.c

```diff
@@ -299,6 +299,26 @@ num_codepoints_for_utf8_bytes(const unsigned char *s, size_t *num_codepoints, si
     return state != UTF8_ACCEPT;
 }
 
+NPY_NO_EXPORT npy_int64
```

Comment:
This function is no longer used! Since we removed the other bits, perhaps this one too?